docs/guide/docs/integrating/existing-backend.md

```diff
@@ -43,25 +44,24 @@ if (event.type === 'update_profile') {
 
 Then send a custom event : `window.botpressWebChat.sendEvent({ type: 'update_profile' })`
 
-## Using a Custom User ID
+## Custom User ID
 
-Users get a new unique User ID each time they use a different device. To offer a consistent user experience across different devices, you may want to provide your customer with the same Botpress User ID to make available past conversations, user data, etc.
+Users get a new unique User ID each time they use a different device. To offer a consistent user experience across other devices, you may want to provide your customer with the same Botpress User ID to ensure past conversations and user data are available.
 
-The `window.botpressWebChat` methods `init` and `configure` both accepts the `userId` parameter. It will override the randomly generated one.
-Since the User ID allows BP to recognize the user and to continue a conversation, these should not be guessable and needs to be unique.
+The `window.botpressWebChat` methods `init` and `configure` both accept the `userId` parameter. It will override the randomly generated one. Since the User ID allows BP to recognize the user and continue a conversation, these should not be guessable and need to be unique for each user.
 
-## Using Custom Components to render special content
+## Custom Components for Content
```

Comment:
I don't think this section should be here. I think custom components are for the webchat, aren't they?